docs/user-tutorial/baseline-generation.md

```diff
@@ -0,0 +1,49 @@
+---
+id: baseline-generation
```

Comment:
please add this page into website/sidebars.js

Reply:
Done.